src/DebugBundle/Connection/ProfilerDatabaseConnection.php

```diff
@@ -315,11 +336,19 @@ public function lastInsertId($seqName = null)
         return $this->connection->lastInsertId($seqName);
     }
 
+    /**
+     * @psalm-suppress InvalidReturnType
+     * @FIXME Parent method returns `mixed`, we should add a `return` statement here.
```

Comment:
This decorator should be synced up with it's underlying implementation / interface. There are a bunch of methods that should have `return` statements and some that shhould not